src/containers/explore-page/components/desktop/CollectionsPage.tsx

```diff
@@ -83,7 +83,6 @@ const CollectionsPage = ({
         setDidLoad={setDidLoadPlaylist}
         key={playlist.playlist_id}
         id={playlist.playlist_id}
-        userId={playlist.playlist_owner_id}
```

Comment:
The card component passes the id prop to the UserBadges component now, so it won't retrieve the correct userId now